bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -124,15 +124,27 @@
     <div class="promo-grid-inner">
       <ul class="promo-grid {% if tweets %}has-twitter-promo{% endif %}">
       {# L10n: Begin home page promos. Line breaks are for visual formatting only. #}
-        <li id="promo-1" class="item promo-large-portrait firefox-android" tabindex="0">
-          <h2 class="primary go">{{ _('Put your Firefox on your Android') }}</h2>
-          <a class="panel-link" href="{{ url('firefox.android.index') }}" data-promotion-id="promo-1" data-promotion-name="Put your Firefox on your Android" data-promotion-class="promo-large-portrait" data-promotion-type="tile">
+      {% if l10n_has_tag('ios_promo') %}
+        <li id="promo-1" class="item promo-large-portrait firefox-ios" data-name="Firefox is now available for iOS" tabindex="0">
+          <h2 class="primary go">{{ _('Firefox is<br> now available<br> for iOS') }}</h2>
+          <a class="panel-link" href="{{ url('firefox.ios') }}">
+            <div class="secondary">
+              <h3>{{ _('Add it to your iPhone, iPad &amp; iPod touch.') }}</h3>
+              <p class="more">{{ _('Get Firefox for iOS') }}</p>
+            </div>
+          </a>
+        </li>
+      {% elif l10n_has_tag('hello_promo') %}
+        <li id="promo-1" class="item promo-large-portrait firefox-hello" data-name="Meet Firefox Hello" tabindex="0">
+          <h2 class="primary go">{{ _('Meet<br> Firefox Hello') }}</h2>
+          <a class="panel-link" href="{{ url('firefox.hello') }}">
             <div class="secondary">
               <h3>{{ _('The features <br>you love. <br>The privacy <br>you trust.') }}</h3>
               <p class="more">{{ _('Learn about <br>Firefox for Android') }}</p>
             </div>
           </a>
         </li>
```

Comment:
The first part of this promo is Hello, the second Android. What the?

If Hello is intended, making sure we're okay with no promo in this spot if both `l10n_has_tag` conditionals happen to be falsy.